example.config.yml

```diff
@@ -64,11 +64,25 @@ configure_local_drush_aliases: true
 
 # Apache VirtualHosts. Add one for each site you are running inside the VM. For
 # multisite deployments, you can point multiple servernames at one documentroot.
+# NOTE: We create separate vhosts for www and non-www URLs so that the
+#       hostsupdater plugin will create host file entries for both.
+# NOTE: We can't create a vhost with a servername of  "www.{{ drupal_domain }}"
```

Comment:
Unfortunately, this breaks some of the intuitive behavior of the config.yml file, since now users will have to change the domain in multiple places.

For now, I'd be okay with users having to manually add in `www.` for the domain, and I'd like to make it so hostsupdater can get all the domains even if they use variables like `{{ varname }}`.